docs/reference.rst

```diff
@@ -697,6 +697,51 @@ No built-in slot update will run and no hook will be executed.
   ``<key>``.
   ``-`` is converted to ``_`` for use as an environment variable name.
 
+.. _log.logger-sections:
+
+**[log.<logger>] sections**
+
+With a logger section, a *RAUC event logger* can be configured.
+The ``<logger>`` suffix determines the internal name of the logger and must be
+unique per ``system.conf``.
+
+``filename``
+  The log file name used for logging.
+  If not absolute path is given, the location is assumed to be relative to the
+  ``data-directory``.
+  Using a relative file name without ``data-directory`` set will cause a
+  configuration error.
+
+``events``
+  Colon-separated list of events to log. Currently supported event types are:
+
+  * ``install`` - Logs start and end of installation
+  * ``boot`` - Logs boot information
+  * ``mark`` - Logs slot marking information
+  * ...
+
+``format``
+  The output format used for the logger. Supported values are
+
+  * ``readable``: readable mutli-line output
+  * ``short``: Single-line readable output
+  * ``json``: single-line JSON output
+  * ``json-pretty``: formatted JSON output
+
+``max-size``
+  Allows to configure a basic log rotation.
+  When given, the logger's log file will be cut-off before reaching
+  the size configured with ``max-size`` and renamed to ``<filename>.0``.
+  Existing rotation file names will be incremented by one.
+  To configure a maximum number of files to keep, see ``max-files``.
+  Values support common suffixes like ``K``, ``M``, ``G``, to ``T``.
+
+``max-files``
+  Configures the maximum number of rotation files to keep.
+  E.g. if set to ``3``, only ``<filename>.0``, ``<filename>.1`` and
+  ``<filename>.2`` will be kept during rotation.
+  Defaults to 10 if unset.
```

Comment:
This is no longer correct.

Reply:
The mentioning of `.0` is invalid, yes. The default of 10 however was just added and should be valid.